Wiz/wiz-audit-logs/ingest/parser.yml

```diff
@@ -99,3 +102,12 @@ stages:
           event.category: ["process"]
           event.type: ["start"]
         filter: "{{json_event.message.action.lower() in ['report', 'rerunreport']}}"
+
+  set_sa_fields:
+    actions:
+      - set:
+          wiz.service_account.creator.email: "{{json_event.message.actionParameters.userEmail}}"
+          wiz.service_account.creator.id: "{{json_event.message.serviceAccount.id}}"
+          wiz.service_account.creator.name: "{{json_event.message.serviceAccount.name}}"
+          service.origin.id: "{{json_event.message.serviceAccount.id}}"
+          service.origin.name: "{{json_event.message.serviceAccount.name}}"
```

Comment:
I'm not sure that these fields are suitable for this information, and we are duplicating the value with the fields upper. I propose to remove them.
```suggestion
```